README-EN.md

```diff
@@ -0,0 +1,48 @@
+# iOS Challenge
+
+The main goal of this challenge is allow us to know better your skills.
+This challenge must be done completely by yourself.
```

Comment:
...must be completely done by yourself.

Reply:
Fixed in ca529bdf5eb31aec656dd6e41df110c39955f2aa.